db/migrate/20171010154443_add_to_type_to_moves.rb

```diff
@@ -0,0 +1,36 @@
+class AddToTypeToMoves < ActiveRecord::Migration[5.0]
+  def up
+    add_column :moves, :to_type, :string
+
+    say_with_time "Evaluating `to_type` in moves..." do
+      moves = Move.all
+
+      moves.each do |move| 
+        unless move.to.nil?
```

Comment:
Not that important as this is run only once, but
items.each do |item|
  unless condition -> next if
    do something
  end
end

Reply:
so you do not indent one more layer and the code is more legible